misc/acrn-config/launch_config/pt.py

```diff
@@ -215,7 +215,7 @@ def gen_pt(names, sel, vmid, config):
 def gen_pt_head(names, sel, vmid, config):
 
     # get passthrough device for specify board and uos
-    cap_pt = launch_cfg_lib.get_board_pt_dev(names, vmid)
+    cap_pt = launch_cfg_lib.get_pt_dev()
```

Comment:
The modification unify the list for different boards/UOS? What if user choose a device that doesn't exist on the using board?

Reply:
The available pass through device which show upon on the webUI are from the board info.
if there is no pass through device for PT, then webUI will show empty which user have not chance to chose.

Reply:
OK, thanks for explanation. However, the comment "_get passthrough device for specify board and uos_" is still valid after the change here?

Reply:
> The available pass through device which show upon on the webUI are from the board info.
> if there is no pass through device for PT, then webUI will show empty which user have not chance to chose.

At the meantime, user will know there is no device on the board if webUI show the empty of the pass through device.

Reply:
> OK, thanks for explanation. However, the comment "_get passthrough device for specify board and uos_" is still valid after the change here?

the comment already remove in the PR.

Reply:
> OK, thanks for explanation. However, the comment "_get passthrough device for specify board and uos_" is still valid after the change here?

i will update the PR, without the comment.